packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -22,9 +22,10 @@ import {
   Variable
 } from '@vscode/debugadapter';
 import { DebugProtocol } from '@vscode/debugprotocol';
-import { MetricError, MetricLaunch } from '..';
+import { MetricError, MetricGeneral, MetricLaunch } from '..';
```

Comment:
Unused imports - were you planning to use them for anything?

Reply:
Oh no, I just missed to remove them in my previous commit.